google/cloud/spanner/internal/connection_impl_test.cc

```diff
@@ -310,25 +312,28 @@ std::unique_ptr<MockStream> MakeReader(
   return reader;
 }
 
-std::unique_ptr<MockStream> MakeReader(
+// Creates a MockStreamingReadRpc from an empty list.
+template <typename ResponseType>
+std::unique_ptr<MockStreamingReadRpc<ResponseType>> MakeReader(
+    std::initializer_list<internal::NonConstructible>,
+    Status status = Status()) {
+  return MakeReader(std::vector<ResponseType>{}, std::move(status));
+}
+
+// Creates a MockStreamingReadRpc from protos in text format.
+template <typename ResponseType>
+std::unique_ptr<MockStreamingReadRpc<ResponseType>> MakeReader(
     std::vector<std::string> const& responses, Status status = Status()) {
-  std::vector<google::spanner::v1::PartialResultSet> response_protos;
-  response_protos.resize(responses.size());
```

Comment:
optional nit: If clang-tidy were smart enough, it would probably yell at us for not `response_protos.reserve(responses.size())` before the loop.

Reply:
(I am noticing I make a lot of comments on behalf of others)

Reply:
Done.